gen-apidocs/generators/config.yaml

```diff
@@ -180,6 +183,9 @@ resource_categories:
       version: v1
       group: core
       description_note: "These are assigned to <a href=\"#pod-v1-core\">Pods</a> using <a href=\"#persistentvolumeclaim-v1-core\">PersistentVolumeClaims</a>."
+    - name: PriorityLevelConfiguration
+      version: v1alpha1
```

Comment:
Would it be possible to auto generate this configuration file?
Why are there only a few `description_note` fields?